pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -830,17 +725,12 @@ func (v *vpnShoot) getVolumeMounts(secrets []vpnSecret) []corev1.VolumeMount {
 		Name:      volumeNameTLSAuth,
 		MountPath: volumeMountPathSecretTLS,
 	})
-	if !v.values.ReversedVPN.Enabled {
-		volumeMounts = append(volumeMounts, corev1.VolumeMount{
-			Name:      volumeNameDH,
```

Comment:
Can we now drop https://github.com/gardener/gardener/blob/55ffc4bac04a2e068a5d87bbbf4c5cdff287bfba/pkg/operation/botanist/component/vpnshoot/vpnshoot.go#L70?

Reply:
Fixed.